### PR TITLE
docs: LDAP service account setup

### PR DESCRIPTION
**Pull request by ibeckermayer**
Adds instructions for setting up an LDAP service account with the minimum required permissions for Teleport to work.

One thing to note is -- This could be somewhat simplified by applying the Group Policy Object and the fine-grained permissions in the PowerShell script to the service account User directly, rather than applying them to a Group and adding the User to the Group. Since this is my first time configuring anything in Active Directory, I elected to go with the standard chain of GPO -> Group -> User to avoid any potential mishap due to a non-standard configuration.

### CODE REVIEW

File: docs/pages/desktop-access/getting-started.mdx
Comment (zmb3) [inline docs/pages/desktop-access/getting-started.mdx:261]: Let's try to stick to the convention of `svc-xxx` for service accounts, and recommend something like `svc-teleport` for this account name.

File: docs/pages/desktop-access/getting-started.mdx
Comment (zmb3) [inline docs/pages/desktop-access/getting-started.mdx:274]: While you're here, care to add a screenshot here?

File: docs/pages/desktop-access/getting-started.mdx
Comment (zmb3) [inline docs/pages/desktop-access/getting-started.mdx:238]: Not sure I would say "First" anymore, since this is step 4.

File: docs/pages/desktop-access/getting-started.mdx
Comment (zmb3) [inline docs/pages/desktop-access/getting-started.mdx:59]: The proper term here is container, not folder.

```suggestion
Now right click the same container you did before, and select `New > Group`. Choose a Group name and ensure that “Global”
```

File: docs/pages/desktop-access/getting-started.mdx
Comment (zmb3) [inline docs/pages/desktop-access/getting-started.mdx:105]:
```suggestion
Then, on your Domain Controller, again open the "Group Policy Management" window. On the left pane, navigate to
```

File: docs/pages/desktop-access/getting-started.mdx
Comment (zmb3) [inline docs/pages/desktop-access/getting-started.mdx:203]:
```suggestion
domain name. For example, for the domain name `teleport.example.com`, the distinguished name representation is `DC=teleport,DC=example,DC=com`. If you aren't sure, you can get the DN by running `(Get-ADDomain).DistinguishedName` in PowerShell.
```

File: docs/pages/desktop-access/getting-started.mdx
Comment (klizhentas) [inline docs/pages/desktop-access/getting-started.mdx:35]: Nit: if you remove: "In theory anymore  account with the ... ;" the sentence will read better.

File: docs/pages/desktop-access/getting-started.mdx
Comment (klizhentas) [inline docs/pages/desktop-access/getting-started.mdx:38]: Nit: Create a user and a group?

File: docs/pages/desktop-access/getting-started.mdx
Comment (klizhentas) [inline docs/pages/desktop-access/getting-started.mdx:44]: Users will be concerned about old passwords that are insecure and unexpiring? Is there any way to make sure that these passwords are never used

File: docs/pages/desktop-access/getting-started.mdx
Comment (klizhentas) [inline docs/pages/desktop-access/getting-started.mdx:59]: Observation: users will be making a lot of mistakes here. Can we implement something like https://github.com/gravitational/teleport/pull/8896 for Desktop Access?

**Review by klizhentas — APPROVED** [review]

File: docs/pages/desktop-access/getting-started.mdx
Comment (zmb3) [reply docs/pages/desktop-access/getting-started.mdx:44]: This password is used by teleport to talk to AD, but we disable interactive login and restrict the permissions so it can't be used to login to a host or do any actions outside of what teleport requires.

File: docs/pages/desktop-access/getting-started.mdx
Comment (zmb3) [reply docs/pages/desktop-access/getting-started.mdx:59]: We have a separate issue filed to look into how much of this we can automate. It's looking like the answer is "not very much" but we'll see what we can do.

**Comment by ibeckermayer** [comment]
This requires a further update once https://github.com/gravitational/teleport/pull/8932 is merged, so I'm converting it to a draft until that happens.

**Comment by russjones** [comment]
@quinqu Can you take a look at this PR. It looks like Sasha approved, but then Isaiah re-requested a review. I think in that case, the check workflow should fail, but here it's passing.

File: docs/pages/desktop-access/getting-started.mdx
Comment (benarent) [inline docs/pages/desktop-access/getting-started.mdx:284]:
```suggestion
    # address will use port 636, like: domain-controller.example.com:636.
```

File: docs/pages/desktop-access/getting-started.mdx
Comment (benarent) [inline docs/pages/desktop-access/getting-started.mdx:257]: This order is much better.

File: docs/pages/desktop-access/getting-started.mdx
Comment (benarent) [inline docs/pages/desktop-access/getting-started.mdx:221]: In this case, the cluster name needs to be same as the FQDN of the Teleport proxy?

File: docs/pages/desktop-access/getting-started.mdx
Comment (benarent) [inline docs/pages/desktop-access/getting-started.mdx:227]:
```suggestion
dsacls "CN=TELEPORT,CN=CDP,CN=Public Key Services,CN=Services,CN=Configuration,$DomainDN" /I:T /G "$($TeleportLDAPGroupName):CCDC;cRLDistributionPoint;"
```

File: docs/pages/desktop-access/getting-started.mdx
Comment (benarent) [inline docs/pages/desktop-access/getting-started.mdx:225]:
```suggestion
dsacls "CN=CDP,CN=Public Key Services,CN=Services,CN=Configuration,$DomainDN" /I:T /G "$($TeleportLDAPGroupName):CC;container;"
```

File: docs/pages/desktop-access/getting-started.mdx
Comment (benarent) [inline docs/pages/desktop-access/getting-started.mdx:229]:
```suggestion
dsacls "CN=TELEPORT,CN=CDP,CN=Public Key Services,CN=Services,CN=Configuration,$DomainDN " /I:T /G "$($TeleportLDAPGroupName):WP;certificateRevocationList;"
```

**Comment by benarent** [comment]
Good job improving the setup instructions @ibeckermayer, I've found a few small issue and had submitted suggestions. 

Is this ready to be deployed? I'm waiting for these instrustions before recording my setup video.

File: docs/pages/desktop-access/getting-started.mdx
Comment (ibeckermayer) [reply docs/pages/desktop-access/getting-started.mdx:221]: No, that should say "Domain Controller domain"

**Comment by ibeckermayer** [comment]
@benarent 

> Is this ready to be deployed? I'm waiting for these instrustions before recording my setup video.

It's ready in the sense that these instructions are complete. I believe I need at least one more code owner approval and it looks like there are some new CI checks that are failing, but I don't know why (@quinqu ?)

**Review by zmb3 — APPROVED** [review]

**Comment by ibeckermayer** [comment]
FTR: The failing checks are under development and are optional, per https://gravitational.slack.com/archives/C0DF0TPMY/p1637295406278500

**Pull request merged**
